src/Explorer/Controls/Settings/SettingsComponent.tsx

```diff
@@ -140,6 +145,7 @@ export class SettingsComponent extends React.Component<SettingsComponentProps, S
       this.offer = this.collection?.offer();
       this.isAnalyticalStorageEnabled = !!this.collection?.analyticalStorageTtl();
       this.shouldShowIndexingPolicyEditor = userContext.apiType !== "Cassandra" && userContext.apiType !== "Mongo";
+      this.shouldShowPartitionKeyEditor = userContext.apiType === "SQL";
```

Comment:
We should check with the DTS folks whether this is available in Mooncake/Fairfax/AGCs